ibis/backends/dask/udf.py

```diff
@@ -116,10 +125,25 @@ def lazy_agg(*series: pandas.Series):
         # Depending on the type of operation, lazy_result is a Delayed that
         # could become a dd.Series or a dd.core.Scalar
         if isinstance(op, ops.AnalyticVectorizedUDF):
-            meta = pandas.Series(
-                [], name=args[0].name, dtype=op._output_type.to_dask()
-            )
+            if isinstance(op._output_type, dt.Struct):
+                meta = make_struct_op_meta(op)
+            else:
+                meta = make_meta_series(
+                    dtype=op._output_type.to_dask(), name=args[0].name,
+                )
             result = dd.from_delayed(lazy_result, meta=meta)
+
+            if args[0].known_divisions:
+                # result is going to be a single partitioned thing,
+                # but we need it to match incoming data so we can dd.concat
+                # downstream. However we know that this udf did not change
+                # the index.
+                original_divisions = args[0].divisions
```

Comment:
Probably good idea to validate all divisions are the same for all args, other wise I don't know what the behavior would be.

Reply:
(More of a defensive check)